images/ubuntu/scripts/build/install-azure-cli.sh

```diff
@@ -4,10 +4,30 @@
 ##  Desc:  Install Azure CLI (az)
 ################################################################################
 
+source $HELPER_SCRIPTS/etc-environment.sh
+# AZURE_EXTENSION_DIR shell variable defines where modules are installed
+# https://docs.microsoft.com/en-us/cli/azure/azure-cli-extensions-overview
+export AZURE_EXTENSION_DIR=/opt/az/azcliextensions
+setEtcEnvironmentVariable "AZURE_EXTENSION_DIR" "$AZURE_EXTENSION_DIR"
```

Comment:
This PR seems to have changed the convention away from the `setEtcEnvironmentVariable` call back to `| tee -a /etc/environment`. I'd be happy to change these calls either way.

Reply:
@mikhailkoliada your thoughts? I'd like to get this closed with the pace at which you folks are updating files, I need to rebase this PR every other day at the moment ;)>